src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -500,19 +500,19 @@
                     }
                     else
                     {
-                        var diffRevisionForButton = Model.ReviewContent.APIRevisions[diffRevisionTypeFirst];
+                        var diffRevisionForButton = Model.ReviewContent.APIRevisionsGrouped[diffRevisionTypeFirst].OrderByDescending(c => c.CreatedOn);
                         var urlValue = @Url.ActionLink("Review", "Assemblies", new
                         {
                             id = @Model.ReviewContent.Review.Id,
-                            revisionId = @Model.ReviewContent.ActiveRevision.Id,
+                            revisionId = @Model.ReviewContent.ActiveAPIRevision.Id,
                             diffRevisionId = diffRevisionForButton.First().Id,
                             doc = @Model.ShowDocumentation,
                             diffOnly = @Model.ShowDiffOnly
                         });
                         <a class="input-group-text" href="@urlValue">Diff:</a>
                     }
-                    <select id="diff-revision-type-select" aria-label="Revision Type Select">
-                        @foreach (var apiRevisionType in Model.ReviewContent.APIRevisions.Keys)
+                    <select id="diff-revision-type-select" aria-label="Revision Type Select">
+                        @foreach (var apiRevisionType in Model.ReviewContent.APIRevisionsGrouped.Keys.Order())

```

Comment:
how is this order applied? to aggregate by type?

Reply:
Its just to ensure its in alphabetical order meaning Automatic will always be first.